tpp/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@token-io/tpp",
-    "version": "1.0.16",
+    "version": "1.0.17",
```

Comment:
I think you need to bump to 18 because bulkTransfer (being merged tmrw) is 16, also, if core is updated, both tpp and app needs version bump, so app should be bumped as well.

Reply:
also app is also bumped in the bulkTransfer pr as well

Reply:
Do I need to bump app version too (I did not change it, but I changed core)?